dev/devicelab/bin/tasks/drive_perf_debug_warning.dart

```diff
@@ -8,14 +8,16 @@ import 'package:flutter_devicelab/framework/adb.dart';
 import 'package:flutter_devicelab/framework/framework.dart';
 import 'package:flutter_devicelab/framework/utils.dart';
 
-Future<String> _runWithMode(String mode, String deviceId) {
-  return evalFlutter('drive', options: <String>[
+Future<String> _runWithMode(String mode, String deviceId) async {
```

Comment:
I will be out, so don't let this hold too much up - but would it be worth just getting both `stdout` and `stderr`, concatenating them (perhaps with a dividing message) and returning that?

Reply:
I didn't do that because I'm afraid of breaking all the existing tests that may rely on matching the full stdout...